tests/test_animal.py

```diff
@@ -1552,14 +1552,113 @@ def test_lactating_cow_manure_calculations():
     pass
 
 
-def test_optimize():
+def test__get_ration():
+    """Unit test for function _get_ration in file routines/animal/ration/calf_ration.py"""
+    actual = RUFAS.routines.animal.ration.calf_ration.CalfRationManager._get_ration()
+    expected = {'155': 1, '157': 2, 'status': 'Optimal', 'objective': 4.5}
+    assert actual == expected
+
+
+def test_optimize(mocker: MockerFixture):
     """Unit test for function optimize in file routines/animal/ration/calf_ration.py"""
```

Comment:
Right now the docstrings for these tests specify what they are testing but not what they are testing for, which I think they should.
```suggestion
    """Tests the optimize function to verify it returns a correctly optimized calf ration."""
```